### ISSUE TITLE
preview in file browser

### ISSUE THREAD

**Issue by phaabe** [Beep6581/RawTherapee#6900]
**problem**
in the file browser, it's nice to be able to give stars and tag the photos. unfortunately for those decisions the thumbnails are too small.

**solution 1**
enable quick preview by maybe hitting the space bar, showing the full photo instead of the thumbnails. Space bar to go back. 

**solution 2**
Enable skipping through photos in larger representation.

**note**
I am aware that double click shows the full photo in the editor, but for quick tagging there is no need to load the full editor which takes always a second on my computer.

I'm glad if there is another way of solving this. If not I vote for an implementation of something similar to the two solutions.

**Comment by marter001** [comment]
Are you aware of the "inspector" on the right tab? If set in preferences, you can open inspector in full screen.

**Comment by phaabe** [comment]
> Are you aware of the "inspector" on the right tab? If set in preferences, you can open inspector in full screen.

yes, but it only shows a zoomed-in view. It's fast an snappy, but I cannot zoom out to see the picture as a hole and big. 
Or am I missing something?

**Comment by marter001** [comment]
when using in a separate window, you can use the mouse-wheel to zoom in and out.

**Comment by Lawrence37** [comment]
Already implemented by #5593.

**Comment by phaabe** [comment]
I came this far (if anyone stumbles on my issue):

- This feature comes with 5.9 (via apt I only got 5.8.3 atm).
- After having activated the separate window feature in the settings, I can view a full image in the file browser press+holding the f key.

Note: Unfortunately I still don't feel amazed by this. I will search more in the issues and the wiki. But I need a workflow where I can skip through full size images and apply rankings. Maybe I am looking at the wrong tool as RawTherapee is more focused on delivering Raw Editing.
